src/common/components/AddTodoBtn/AddTodoBtn.tsx

```diff
@@ -12,7 +13,7 @@ const AddTodoBtn: React.FC = () => {
   return (
     <div>
```

Comment:
if we have only one child inside of return statement, it's not necessary to wrap it into the **div**